packages/connectors/src/coinbaseWallet.ts

```diff
@@ -122,7 +100,6 @@ export function coinbaseWallet(parameters: CoinbaseWalletParameters) {
       }
 
       provider.disconnect()
-      provider.close()
```

Comment:
what if `provider`'s version is 3.x?

Reply:
good catch, addressed in [886fb51](https://github.com/wevm/wagmi/pull/3784/commits/886fb51fd306b14e81ec5b1d3afeac2d00a42c15)